lib/src/elastic_client_impl.dart

```diff
@@ -51,6 +51,36 @@ class Client {
     return rs.statusCode == 200;
   }
 
+  Future<List<Alias>> getAliases({List<String> aliases = const []}) async {
+    final rs = await _transport.send(
+      Request(
+        'GET',
+        ['_cat', 'aliases', aliases.join(',')],
+        params: {'format': 'json'},
+      ),
+    );
+    if (rs.statusCode != 200) {
+      throw Exception(
+          'Unable to get aliases information with ${aliases}. ${rs.statusCode} ${rs.body}');
+    }
+    final body = convert.json.decode(rs.body) as List;
+    return body
+        .map(
+          (alias) => Alias(
+            alias: alias['alias'] as String,
+            index: alias['index'] as String,
+            indexRouting: alias['routing.index'] == '-'
```

Comment:
One more question: why is it better to convert this to empty string instead of using the ES-reported `-`?

Reply:
If there is a good reason for it, please add a comment to the code, otherwise let's just keep the original value.

Reply:
Elasticsearch returns `-` as it means there are no index routing or search routing. This is just my guess but `_cat` endpoints are mainly for human, so for better human readability ES return `-` instead of empty string.

I looked up Ruby implementations and it looks it just returns raw response,
https://github.com/elastic/elasticsearch-ruby/blob/master/elasticsearch-api/lib/elasticsearch/api/actions/cat/aliases.rb#L53

Following ruby impl I will leave these parameters, will update my code pretty soon!

Reply:
I just updated my code. Thanks for your advice! :smile: